data/themes/default/style.css

```diff
@@ -620,15 +620,17 @@ TrackContentObjectView   {
 	qproperty-selectedColor: #006B65;
 	qproperty-BBPatternBackground: #373d48;
 	qproperty-textColor: #fff;
+	qproperty-textBackgroundColor: rgba(0, 0, 0, 75);
 	qproperty-textShadowColor: rgb(0,0,0,200);
```

Comment:
With shadows:

![3704-colored-bb-patterns-shadowed](https://user-images.githubusercontent.com/9293269/28337728-e560789a-6c06-11e7-9c55-ff0a42f95994.png)

For my taste it looks cleaner without the shadow. It might also make sense to investigate whether it would make sense to use [QGraphicsDropShadowEffect](https://doc.qt.io/qt-4.8/qgraphicsdropshadoweffect.html) to render the shadow. But that would be something for another pull request.